src/test/bin/winkernel/driver.cpp

```diff
@@ -68,7 +68,9 @@ void __cdecl operator delete[] (_In_opt_ void* Mem) {
     }
 }
 
+#ifndef QUIC_BUILD_STATIC
```

Comment:
This change actually isn't necessary, because we will never support kernel mode statically built.